Assets/Package/Runtime/Console/ConsoleManager.cs

```diff
@@ -259,6 +259,37 @@ public void LoadHistory()
             InputHistory.LoadHistory();
         }
 
+        void IAutocompleteOptionProvider.GetParemeterAutocompleteOptions(string[] words, int wordIndex, IList<string> optionList)
+        {
+            EnsureInitialized();
+
+            if (wordIndex == 0)
+            {
+                foreach (var commandName in RegisteredMethods.Keys.ToArray())
+                {
+                    if (optionList.Contains(commandName))
+                    {
+                        continue;
+                    }
+
+                    optionList.Add(commandName);
+                }
+            }
+            else
+            {
+                if (!namesToInstances.ContainsKey(words[0]))
+                {
+                    return;
+                }
+
+                var command = namesToInstances[words[0]];
```

Comment:
W dwóch miejscach bierzesz words[0] więc warto to zcachować przed ifem i użyć zamiast dwa razy indekser tablicy wywoływać.